freqtrade/rpc/api_server/ws/proxy.py

```diff
@@ -15,6 +15,10 @@ class WebSocketProxy:
     def __init__(self, websocket: WebSocketType):
         self._websocket: Union[FastAPIWebSocket, WebSocket] = websocket
 
+    @property
+    def raw(self):
```

Comment:
i find `.raw` quite generic, and it's completely unclear that it's going to return the websocket object - maybe `.raw_websocket` or something might be better...

Reply:
Understood, I had the same thought originally.